tests/test_data/aldi-sued.de/aldisued.json

```
@@ -0,0 +1,62 @@
+  "site_name": "ALDI SÜD",
+  "host": "aldi-sued.de",
+  "language": "de",
+  "title": "Fränkische Weißweinsuppe",
+  "ingredients": [
+    "1 Bund Suppengemüse (alternativ 2 Möhren, 150 g Knollensellerie, 1/2 Stange Porree)",
+    "1 Zwiebel",
+    "4 EL MILSANI Butter",
+    "2 EL GOLDÄHREN Qualitäts-Weizenmehl Type 405",
+    "500 ml trockener Weißwein",
+    "500 ml LE GUSTO Klare Brühen im Glas, Sorte: Gemüse",
+    "2 Lorbeerblätter (optional)",
+    "200 g MILSANI Sahne",
+    "LE GUSTO Salz",
+    "LE GUSTO Pfeffer",
+    "LE GUSTO Muskat",
+    "DIADEM Zucker",
+    "3 Scheiben GOLDÄHREN Sandwichtoast",
+    "1 Prise LE GUSTO Zimt",
+    "3 Zweige Bio-Topfkräuter Petersilie"
+  ],
+  "ingredient_groups": [
+    {
+      "ingredients": [
+        "1 Bund Suppengemüse (alternativ 2 Möhren, 150 g Knollensellerie, 1/2 Stange Porree)",
+        "1 Zwiebel",
+        "4 EL MILSANI Butter",
+        "2 EL GOLDÄHREN Qualitäts-Weizenmehl Type 405",
+        "500 ml trockener Weißwein",
+        "500 ml LE GUSTO Klare Brühen im Glas, Sorte: Gemüse",
+        "2 Lorbeerblätter (optional)",
+        "200 g MILSANI Sahne",
+        "LE GUSTO Salz",
+        "LE GUSTO Pfeffer",
+        "LE GUSTO Muskat",
+        "DIADEM Zucker",
+        "3 Scheiben GOLDÄHREN Sandwichtoast",
+        "1 Prise LE GUSTO Zimt",
+        "3 Zweige Bio-Topfkräuter Petersilie"
+      ]
+    }
+  ],
+  "instructions": "Das Suppengemüse und die Zwiebel schälen und würfeln.\n2 EL Butter in einem Topf schmelzen und das Gemüse darin andünsten. Das Mehl einstreuen und kurz anschwitzen.\nDen Weißwein und die Brühe eingießen. Ggf. Lorbeerblätter hineingeben. Aufkochen und ca. 20 Minuten köcheln lassen.\nDie Suppe durch ein Sieb streichen und zurück in den Topf geben. Die Sahne eingießen und mit Salz, Pfeffer, Muskat und Zucker abschmecken.\nDas Toastbrot würfeln. 2 EL Butter schmelzen und die Brotwürfel darin goldbraun rösten. Mit Zimt bestreuen.\nDie Petersilie waschen und grob hacken. Die Weinsuppe mit den Zimt-Croûtons und der Petersilie anrichten und servieren.",
+  "instructions_list": [
+    "Das Suppengemüse und die Zwiebel schälen und würfeln.",
+    "2 EL Butter in einem Topf schmelzen und das Gemüse darin andünsten. Das Mehl einstreuen und kurz anschwitzen.",
+    "Den Weißwein und die Brühe eingießen. Ggf. Lorbeerblätter hineingeben. Aufkochen und ca. 20 Minuten köcheln lassen.",
+    "Die Suppe durch ein Sieb streichen und zurück in den Topf geben. Die Sahne eingießen und mit Salz, Pfeffer, Muskat und Zucker abschmecken.",
+    "Das Toastbrot würfeln. 2 EL Butter schmelzen und die Brotwürfel darin goldbraun rösten. Mit Zimt bestreuen.",
+    "Die Petersilie waschen und grob hacken. Die Weinsuppe mit den Zimt-Croûtons und der Petersilie anrichten und servieren."
+  ],
+  "description": "Eine fränkische Tradition: Die aromatische Weißweinsuppe wird mit duftenden Zimt-Croûtons serviert.",
+  "total_time": 35,
+  "cook_time": 20,
+  "prep_time": 15,
+  "ratings": 0.0,
```

```suggestion
  "ratings": 0,
```

I briefly wondered whether this indicates a bug in our library -- returning a zero rating count for a recipe -- but no, that information is genuinely present in the webpage, and the fact that there were (currently) no ratings for a recipe is valid information.

Even so, let's assert on the value being `0` integer rather than floating-point.